server/controller.ts

```diff
@@ -2,7 +2,9 @@ import joi from "joi";
 import { Request, Response } from "express";
 
 import * as service from "./service";
-import { logger } from "./index";
+import path from "path";
```

Comment:
Well tried but this implementation introduces even more repetitive code. The trick is to either parse it from stack trace (some examples on [SO](https://stackoverflow.com/questions/14172455/get-name-and-line-of-calling-function-in-node-js)) or use something like https://github.com/felixge/node-stack-trace